jupyter_book/default_config.yml

```diff
@@ -34,6 +46,11 @@ html:
     hypothesis              : false
     utterances              : false
 
+#######################################################################################
+# LaTeX-specific settings
+latex:
+  latex_engine              : pdflatex
```

Comment:
are we sticking with pdflatex or xelatex? xelatex is a bit slower, but has support for wider range of characters which can come handy.

Reply:
Yeh I would say xelatex. Is it always packaged with TeX distributions though?
We can also add a comment string here listing the alternatives: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-latex_engine

Reply:
cc: @mmcky

Reply:
`pdflatex` is the default used by `sphinx` so I imagine most testing in `sphinx` is against `pdflatex` (as the default option). We always use `xelatex` on `quantecon` projects -- and is a good default choice for more scientific book. But there is also `lualatex` etc. 

It isn't hard to set in `_config.yml` particularly if we setup the default value so that `jb create` populates with an easy to change setting. 

Essentially I don't really mind either way -- I think `pdflatex` isn't a bad "industry standard" but I can't think of a case where building with `xelatex` will be any worse off than `pdflatex`.

Reply:
re: testing -- come to think of it most tests are probably just against `tex` output so there may not be a lot of testing against the pdf conversion method.

Reply:
We could just set it to None, to use the sphinx default, then yeh add a commment with the main options

Reply:
@chrisjsewell yes it is available with all the major TeX distributions like TeX Live, MikTex, MacTex, Lyx. Not sure if any other distributions are common though.  https://en.wikipedia.org/wiki/XeTeX .
Hmm. True, we can list the alternatives, but hopefully, the theme we create for LaTex does not behave weirdly in other alternatives. Only time can tell.

Reply:
theme support is a good point @AakashGfude. Lets visit that in #586 

@chrisjsewell I don't see a downside in keeping it `pdflatex` rather than `None` as it is more descriptive and explicit in terms of what is being used.

Reply:
Fair, I don't see it as a big issue, although I do note that sphinx use xelatex as the default for chinese and greek.

Reply:
I guess in summary I think `pdflatex` is a pretty sane default value  -- despite the fact that I will need to update it to `xelatex` for all our projects `:-)`. I think "most" users of `tex` interface with `pdflatex` by default -- but a good writeup of options will be really useful.

Reply:
OK - well then let's be brave and set `xelatex` for built-in unicode support. It is getting more and more popular.

Reply:
I might do this in a separate PR, since I have a feeling it may break the CI
(I've added a comment to the default_config with the options)

Reply:
Just want to say that there are people out there (e.g. me) for whom `pdflatex` doesn't cut it because of its poor support for Unicode.